specification/redis/resource-manager/Microsoft.Cache/stable/2022-06-01/redis.json

```diff
@@ -2666,6 +2666,16 @@
             "name": "ReplicationRole",
             "modelAsString": false
           }
+        },
+        "geoReplicatedPrimaryHostName": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Fully qualified geo subdomain URL which points to geo-primary cache among the linked redis caches."
```

Comment:
Is this really an URL, or just a hostname/DNS name? As seen in the examples, is not really an URL since it has no scheme such as https://, and does not specify a usable redis port number.

Reply:
Updated description.